app/client/src/modules/ui-builder/ui/wds/WDSMenuButtonWidget/widget/index.tsx

```
@@ -1,81 +1,61 @@
 
 class WDSMenuButtonWidget extends BaseWidget<
   MenuButtonWidgetProps,
   WidgetState
 > {
   constructor(props: MenuButtonWidgetProps) {
     super(props);
-
-    this.state = {
-      isLoading: false,
-    };
   }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)